datanymizer_engine/src/uniq_collector.rs

```diff
@@ -1,10 +1,16 @@
 use once_cell::sync::Lazy;
-use std::{collections::HashSet, sync::Mutex};
-static GLOBAL_DATA: Lazy<Mutex<HashSet<String>>> = Lazy::new(|| Mutex::new(HashSet::new()));
+use std::collections::{hash_map::DefaultHasher, HashSet};
+use std::hash::{Hash, Hasher};
+use std::sync::Mutex;
 
-pub(crate) fn add_to_collector(item: String) -> bool {
+static GLOBAL_DATA: Lazy<Mutex<HashSet<u64>>> = Lazy::new(|| Mutex::new(HashSet::new()));
+
+// Should we move this to `transformer` module?
```

Comment:
No, I guess... Let's encapsulate the logic for managing the uniqueness values a separate module.